common/src/chain/transaction/transaction_index.rs

```diff
@@ -188,8 +230,14 @@ mod tests {
         );
         assert_eq!(tx_index.get_output_count(), 3);
 
+        let p = if let SpendablePosition::Transaction(ref p) = tx_index.position {
+            p
+        } else {
+            unreachable!();
+        };
```

Comment:
Very minor comment: I'd personally go for a `match` expression instead of `if let` here. But that's just a style preference

Reply:
Yeah, it's just a test. That's why I don't care. Usually I prefer match as well.